### ISSUE TITLE
any real tutorial for this available?

### ISSUE THREAD

**Issue by WorthyPro** [Deimos/SubredditSimulator#4]
hi im trying to make a subreddit simulator of my own but its quite difficult to get this up and running.
is there any tutorial(via a yt video or just text) to make this thing.
tia

**Comment by gbionescu** [comment]
Not really. But you can backtrack problems when setting up the DB and things will work fine.
